src/tasks/wifi.c

```diff
@@ -419,6 +437,13 @@ static void process_new_connection(struct Serial *s)
 	esp8266_drv_close(s);
 }
 
+static bool is_valid_ipv4(const char *address)
+{
+        return NULL != address &&
```

Comment:
Couldn't this be simplified to `return address && *address && !(STR_EQ(address, IPV4_NO_IP_STR));`